core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -1655,7 +1657,8 @@ protected void restoreTabs(@Nullable String zimArticles, @Nullable String zimPos
         }
       }
       selectTab(currentTab);
-      webViewList.get(currentTab).loadUrl(UpdateUtils.reformatProviderUrl(urls.getString(currentTab)));
+      webViewList.get(currentTab)
```

Comment:
is this just formatting? 
(small nit: please stash formatting changes like this + the reordering of the imports so it's clearly visible what the actual changes are) :)

Reply:
Just formatting. When should formatting changes be done? As a separate ticket, only by the static analysis, or never? Assuming that the formatting is correct. I have bound formatting and import optimization to ctrl+s, which I spam whenever. This means I'd have to stash the changes whenever I am working on any file that is not formatted according to our code style.